fink_science/image_classification/README.md

```diff
@@ -0,0 +1,39 @@
+# Image Classification
+
+This module perform a classification of alerts based on image visual content. It use only the cutoutScience image and return a single string label which describe image alert. 
+
+# Procedure
+
+The image classification perform in three steps :
+* Filter images that are noised and/or corrupted (corrupted image contains Nan based on python definition of Nan)
+
+* All images that are not noised and not corrupted are binarize with a threshold computed by triangle_thershold method from scikit_image library
+then binary image which contains only small region are categorize as star else they go to the third step
```

Comment:
Typo: All images that are not **noisy** and not corrupted are **binarized** with a threshold computed by **the triangle_threshold** method from scikit_image library then binary **images** which **contain** only small **regions** are **categorized** as `star` else they go to the third step